src/resolvers/partials/mutation.handlebars

```diff
@@ -0,0 +1,5 @@
+Mutation: {
```

Comment:
This split do not make much sense. 
Would we ever need to modify this? Probably not. 
We can have implementation doing all and not using handlebars at all.